pcsx2/gui/Dialogs/ConvertMemoryCardDialog.cpp

```diff
@@ -84,15 +84,18 @@ Dialogs::ConvertMemoryCardDialog::ConvertMemoryCardDialog( wxWindow* parent, con
 void Dialogs::ConvertMemoryCardDialog::CreateControls( const MemoryCardType sourceType ) {
 	m_text_filenameInput = new wxTextCtrl( this, wxID_ANY, wxEmptyString, wxDefaultPosition, wxDefaultSize, wxTE_PROCESS_ENTER );
 
-	RadioPanelItem toFile8MB = RadioPanelItem( _( "8MB File" ), pxE( L"Convert this memory card to a standard 8 MB Memory Card .ps2 file." ) )
+	RadioPanelItem toFile8MB = RadioPanelItem( _( "8 MB File (most compatible)" ), pxE( L"Convert this memory card to a standard 8 MB Memory Card .ps2 file." ) )
 		.SetInt( MemoryCardConversionType::MemoryCardConversion_File_8MB );
-	RadioPanelItem toFile16MB = RadioPanelItem( _( "16MB File" ), pxE( L"Convert this memory card to a 16 MB Memory Card .ps2 file." ) )
+	RadioPanelItem toFile16MB = RadioPanelItem( _( "16 MB File" ), pxE( L"Convert this memory card to a 16 MB Memory Card .ps2 file." ) )
 		.SetInt( MemoryCardConversionType::MemoryCardConversion_File_16MB );
-	RadioPanelItem toFile32MB = RadioPanelItem( _( "32MB File" ), pxE( L"Convert this memory card to a 32 MB Memory Card .ps2 file." ) )
+	RadioPanelItem toFile32MB = RadioPanelItem( _( "32 MB File" ), pxE( L"Convert this memory card to a 32 MB Memory Card .ps2 file." ) )
 		.SetInt( MemoryCardConversionType::MemoryCardConversion_File_32MB );
-	RadioPanelItem toFile64MB = RadioPanelItem( _( "64MB File" ), pxE( L"Convert this memory card to a 64 MB Memory Card .ps2 file." ) )
+	RadioPanelItem toFile64MB = RadioPanelItem( _( "64 MB File" ), pxE( L"Convert this memory card to a 64 MB Memory Card .ps2 file." ) )
 		.SetInt( MemoryCardConversionType::MemoryCardConversion_File_64MB );
-	RadioPanelItem toFolder = RadioPanelItem( _( "Folder" ), _( "Convert this memory card to a folder of individual saves." ) )
+	RadioPanelItem toFolder = RadioPanelItem( _( "Folder" ), _(
+		"Convert this memory card to a folder of individual saves. "
+		"Choose this to go beyond the save-limits of 8-64 MB cards and if you only use PCSX2. "
```

Comment:
> "Choose this to go beyond the save-limits of 8-64 MB cards and if you only use PCSX2

This does work but I think some of the phrasing is a bit ambiguous. The major points to highlight here should be

1. There is no maximum capacity. Don't worry about saying it "goes beyond what file memcards do", it would be better to just tell it how it is, "No capacity limit".
2. Folder memcards are exclusive to PCSX2. "If you only use PCSX2" is ok, but I am personally biased towards "Not compatible with other PS2 emulators.
3. One selling point of folder memcards is that they make it easy to directly access the save files the game uses. I would consider adding a bit about that, maybe something like "Allows direct access to individual save files".
4. I think "You can always convert back" is pointless; it's already implied since the user is already converting.